src/Api.php

```diff
@@ -223,6 +234,21 @@ public function purgeUrl($url = '') {
   }
 
   /**
+   * Performs an actual purge request for the given path.
+   */
+  protected function purgeQuery($path) {
+    drupal_http_request(url($path, ['absolute' => TRUE]), [
```

Comment:
`drupal_http_request ` is not part of the Drupal 8 API : https://api.drupal.org/api/drupal/includes%21common.inc/function/drupal_http_request/7.x

Reply:
That's something that was already there. I only added $this->_webhook->sendWebHook('Purged ' . $path, "purge_keys"); 
purgeQuery is not actually used anywhere as I see, so I'll just remove it.